tutorials/interactive-example/README.md

```diff
@@ -0,0 +1,8 @@
+# Interactive Example
+
+The interactive example in `examples/interactive` can be run by Linux users. It is not yet supported for MacOS users.   
+
+To run the example, run the following commands:
+1. Navigate to the `examples/interactive` directory.
+2. Comment [this line](https://github.com/thanos-io/thanos/blob/bd134d7a823708fa135e7a6931e76f581be5f879/examples/interactive/interactive_test.go#L92) in the file `interactive_test.go`.   
```

Comment:
Can we just comment this line by default? @metonymic-smokey

Reply:
Then if someone runs `go test ./...` the interactive example will trigger as well, which is not a good idea I think.